aloscene/frame.py

```diff
@@ -168,6 +174,19 @@ def append_boxes2d(self, boxes: BoundingBoxes2D, name: str = None):
         """
         self._append_label("boxes2d", boxes, name)
 
+    def append_points2d(self, points: Points2D, name: str = None):
+        """Attach a set of points to the frame.
+
+        Parameters
+        ----------
+        boxes: Points2D
+            Points to attached to the Frame
+        name: str
+            If none, the points will be attached without name (if possible). Otherwise if no other unnamed
+            points are attached to the frame, the points will be added to the set of points.
+        """
```

Comment:
```suggestion
            Points to attach to the Frame
        name: str
            If None, the points will be attached without name (if possible). Otherwise if no other unnamed
            points are attached to the frame, the points will be added to the set of points.
        """
```